velox/common/caching/SsdFile.h

```diff
@@ -443,7 +442,7 @@ class SsdFile {
   // Reads a checkpoint state file and sets 'this' accordingly if read is
```

Comment:
Now since we removed the file argument, then we should update the comment
`Reads the checkpoint state file`
Delete `Return true for successful read.`